peekingduck/pipeline/nodes/draw/image_processor.py

```diff
@@ -23,7 +23,25 @@
 
 
 class Node(AbstractNode):
-    """Node for changing image contrast and brightness"""
+    """Node for changing image contrast and brightness
+
+    The draw image processor node uses a mathematical algorithm to adjust the
```

Comment:
could we talk about this "algorithm", or link to an article that talks about it? I'm okay with it being in the reference as well

Reply:
Added a reference to an article explaining the use and formula for OpenCV.convertScaleAbs().